environment.yml

```diff
@@ -4,17 +4,18 @@ channels:
   - conda-forge
 dependencies:
   - astropy>=4.0
-  - h5py
```

Comment:
Do we not read any hdf5 directly anymore? I know we offloaded a lot of I/O to pyradiosky and obviously pyuvdata requires this just want to make sure. Figure if we are reading them directly we probably want to leave it in the yaml.

Reply:
We don't import it in pyuvsim, we only use it through pyuvdata and pyradiosky. And since it's a required dependency of pyuvdata I didn't think we needed to list it. But I can add it back if you think it's a good idea.

Reply:
nah this works for me if we're not directly importing it :+1: